demos/BMW_demo/bmw_open.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Share the given share to all the parties in the jiff instance.
+ * @function jiff_broadcast
+ * @ignore
+ * @param {module:jiff-client~JIFFClient} jiff - the jiff client instance.
+ * @param {module:jiff-client~JIFFClient#SecretShare} share - the share.
+ * @param {Array} parties - the parties to broadcast the share to.
+ * @param {number|string} op_id - a unique operation id, used to tag outgoing messages.
+ *
+ */
+
+/*
+
+function ooo(jiff,ls) {
```

Comment:
This file looks good. We can remove the commented out functions (as well as any commented out `console.log()` statements. They will still be in the commit history if we ever want to look back at them so it is safe to remove them.